ouroboros-network/protocol-tests/Ouroboros/Network/Protocol/KeepAlive/Examples.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Ouroboros.Network.Protocol.KeepAlive.Examples where
+
+import           Data.List (cycle)
```

Comment:
Is this an unused input?

Reply:
It is.  Hmm, I think I forgot to add `-Wall` (it should fail in CI).

Reply:
Thanks for spotting this, it turns out there are quite a few warnings with `-Wall` in `ouroboros-protocols-tests`.  It deserves its own PR.